.changeset/slow-nails-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@prairielearn/compiled-assets': patch
```

Comment:
Nit: this should probably be a separate entry that specifically discusses how it defines `process.env.NODE_ENV`. That's not Preact-specific.